### ISSUE TITLE
Issue while training LGCN

### ISSUE THREAD

**Issue by redionxhepa** [yl-1993/learn-to-cluster#80]
Hi ! I am training LGCN on some custiom dataset. Training seems to run somehow but I get the following warning.

"/home/redion/redion_files/learn-to-cluster/faceCluster/lib/python3.6/site-packages/sklearn/metrics/_classification.py:1272: UndefinedMetricWarning: Recall is ill-defined and being set to 0.0 due to no true samples. Use `zero_division` parameter to control this behavior.
  _warn_prf(average, modifier, msg_start, len(result))
"
What does this mean ? I get 97 percent accurracy but precision and recall are 0.

**Comment by yl-1993** [comment]
If precision is high but recall is low, it usually means the clustered results consist of many small clusters. In an extreme case, each instance is taken as a clustere. A possible reason may lie in the wrong linkage when building the graph. Checking the built graph or the training loss may help.
